bot/cogs/defcon.py

```diff
@@ -202,17 +197,17 @@ def mod_log(self) -> ModLog:
         self.mod_log.ignore(Event.guild_channel_update, Channels.defcon)
         await self.channel.edit(topic=new_topic)
 
-    def build_defcon_msg(self, change: str, e: Exception = None) -> str:
+    def build_defcon_msg(self, action: Action, e: Exception = None) -> str:
         """
         Build in-channel response string for DEFCON action.
 
         `change` string may be one of the following: ('enabled', 'disabled', 'updated')
```

Comment:
This docstring is no longer accurate. It can probably just be removed altogether.